tests/conftest.py

```diff
@@ -260,12 +258,15 @@ def oracle(
         ],
         {"from": management},
     )
-
+    
+    calculationsYearnVaults = CalculationsYearnVaults.deploy(oracle, {"from": management})
     calculationsIronBank = CalculationsIronBank.deploy(
         unitrollerAddress, oracle, {"from": management}
     )
     oracle.setCalculations(
         [
+            calculationsOverrides,
+            calculationsYearnVaults,
```

Comment:
Add calculationsYearnVaults to calculation path before this was missing previously

Reply:
On the live oracle calculationsYearnVaults comes after curve, not sure if that makes a difference or not